tests/ctl/core/test_config.py

```diff
@@ -147,7 +147,7 @@ def test_config_from_env_vars() -> None:
 )
 @pytest.mark.unit
 def test_database_url_test_mode_disabled() -> None:
-    os.environ["FIDES_TEST_MODE"] = "False"
+    os.environ["FIDES__TEST_MODE"] = "False"
```

Comment:
I see a lot of repeats here...

maybe adding this as a constant to config utils or something? like:

```python
from fides.ctl.core.config.utils import FIDES_TEST_MODE
```

```python
FIDES_TEST_MODE = True if os.getenv("FIDES__TEST_MODE") else False
```

or something like that?

Reply:
I'd rather leave this for now and update it as a separate PR if needs be, it's just a rename of the env var to fit the convention we've already set

Reply:
if we're already touching every instance of it, I don't see a big difference there and think it falls in line with "leave the codebase better than you found it", its a small thing though

but at the very least, let's open an issue for it. This would be a very easy place for someone to change in the future and not realize there's 6 more places to change